keyboards/handwired/digitizer_demo/config.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
```

Comment:
This needs a license header.  

But also, this may be better as a keymap for `handwired/onekey`

Reply:
yes, good call.  I just wired up a promicro onekey board to test with, so I can make this suggested change.